temporarily-unavailable/Pages/Shared/_Layout.cshtml

```diff
@@ -3,8 +3,8 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>NTBS - coming soon</title>
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true">
```

Comment:
I assume this `~` was the source of your problems earlier?

Reply:
No, it was actually the order of lines in startup 😑